kuronometer-core/src/main/scala/com/github/pedrovgs/kuronometer/free/interpreter/api/KuronometerApiClientConfig.scala

```diff
@@ -0,0 +1,10 @@
+package com.github.pedrovgs.kuronometer.free.interpreter.api
+
+case class KuronometerApiClientConfig(
+                                       scheme: String = "https",
```

Comment:
Not sure if IDEA but most sane projects won't accept this format, should introduce scalaformat